src/styles/main.scss

```diff
@@ -1,3 +1,61 @@
+@import "variables";
+
+html {
+  font-family: "Arial", sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc((#{$day-width} * 7) + 10px);
+  gap: $gap-space;
+  margin: 25px auto;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    align-items: center;
+    justify-content: center;
+    background-color: $background-gray;
+
+    width: $day-width;
+    height: $day-width;
+
+    border-width: 1px;
+    border-style: solid;
+    border-color: black;
+
+    transition: transform 0.5s, background-color 0.5s;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
Why change the cursor style on hover?

Reply:
It`s a condition of the task:

On hovering over a cell:
- cursor should become pointer
- The hovered cell has to become pink